### PR TITLE
Support for .NET 6

### PR DESCRIPTION
**Pull request by danatcofo**
* NET 6 Support
* Target a specific project in the `electronize build/start` command using `/dotnet-project <projectName>.csproj`

NOTE: the .NET6 runtime has targeted System.Drawing.Common at windows architectures and will cause warnings.  This should be addressed at some time.

https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only

### CODE REVIEW

**Comment by danatcofo** [comment]
appveyor is failing on net6 not being supported... looking into that.

File: ElectronNET.sln [hunk 1]
```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27130.2027
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ElectronNET.WebApp", "ElectronNET.WebApp\ElectronNET.WebApp.csproj", "{7C048379-401C-4345-B5E7-BE232DEA8157}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ElectronNET.WebApp.NET5", "ElectronNET.WebApp\ElectronNET.WebApp.NET5.csproj", "{7C048379-401C-4345-B5E7-BE232DEA8157}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ElectronNET.WebApp.NET6", "\\Mac\Home\Repos\Electron.NET\ElectronNET.WebApp\ElectronNET.WebApp.NET6.csproj", "{76F72AED-31F4-4289-AFB5-D7ECA84072B8}"
```
Comment (alborozd) [inline ElectronNET.sln]: looks like the path needs to be fixed

```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ElectronNET.WebApp.NET6", "ElectronNET.WebApp\ElectronNET.WebApp.NET6.csproj", "{76F72AED-31F4-4289-AFB5-D7ECA84072B8}"
```

**Comment by robalexclark** [comment]
Does anyone know if this can the merged into main?

**Comment by danatcofo** [comment]
I have this and most of the other PRs merged in a fork branch.  Working fine for me there.

**Comment by GregorBiswanger** [comment]
The next big update is coming soon!

**Comment by MutatedGamer** [comment]
I tried out a local build of the source branch for this PR and noticed a quirk:

If I specify a `aspCoreBackendPort` in my manifest, I am unable to connect to the ASP.NET Core server. `electronize start` logs `Now listening on: http://localhost:<myport>`, but navigating to that URL gives a 404.

Interestingly, if I start with `electronize start /watch`, I am navigate to `http://localhost:<myport>` and get properly redirected to my SpaProxyServerUrl.

I wonder if this is related: https://github.com/dotnet/aspnetcore/issues/38185

**Comment by danatcofo** [comment]
Yah I didn't implement that functionality into the change set.  I haven't even figured out or researched how to do that yet.

**Pull request merged**